azure-test/tests/azure_app_service_web_app/test-list-query.sql

```diff
@@ -1,3 +1,3 @@
-select id, name
+select id, name, identity
 from azure.azure_app_service_web_app
-where name = '{{resourceName}}'
+where akas::text = '["{{output.resource_aka.value}}", "{{output.resource_aka_lower.value}}"]';
```

Comment:
Do we need to match exact `akas` for LIST call? Can't we use `name` instead, though it will also perform the LIST only?